tools/doc/html.js

```diff
@@ -107,6 +120,10 @@ function render(lexed, filename, template, cb) {
   });
 }
 
+function gaScriptContent() {
+  return fs.readFileSync(gaScriptPath).toString();
```

Comment:
Allowed my self to use `Sync` here as this is a build script, where doing async doesn't do too much for performance, other than increasing code complexity.